### ISSUE TITLE
Test Accessibility Issue Submitted by User

### ISSUE THREAD

**Issue by openstaxalina** [openstax/webview#2193]
Feedback received from: 
Michaela D. Willi Hooper
OER/Textbook Affordability Librarian
Linn-Benton Community College Library
willihm@linnbenton.edu

Other content was simply labeled as "clickable" or "link," and then when clicked, the external material was not accessible (for example, the link in the concepts in action box in [section 1.1](https://cnx.org/contents/s8Hh0oOc@14.1:-Yx1m5Xz@12/1-1-Themes-and-Concepts-of-Biology) of Concepts of Biology)

Would it be possible to test this and provide feedback below?

**Comment by Stackblocks** [comment]
I'm not sure I understand. The link goes to a Wikimedia Commons video which is outside of the scope of our accessibility efforts, barring us including a description of the content in our book with the link.

**Comment by openstaxalina** [comment]
@Stackblocks Ah, I see.  I am going to close this since this is not a technical issue.
